### PR TITLE
Added basic UI

### PR DESCRIPTION
**Pull request by jonahwilliams**
screen shot 2016-09-18 at 1 44 47 pm

### CODE REVIEW

File: modules/state/lib/state.dart [hunk 1]
```diff
@@ -86,6 +86,10 @@ class Board {
     }
   }
 
+  /// Returns a flat list over the values of the board.
```
Comment (matanlurey) [inline modules/state/lib/state.dart]: Consider calling this List<Space> toValues() to make it clear we are creating a new instance every time (i.e. don't use this in a tight loop)

File: modules/ui/web/main.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import 'package:tic_tac_toe.ui/ui.dart';
+import 'package:tic_tac_toe.state/state.dart';
+import 'dart:html';
+import 'dart:async';
+
+void main() {
+  /// An example of an app that is wired up to itself with turn switching.
```
Comment (matanlurey) [inline modules/ui/web/main.dart]: This should go on the previous line :)

File: modules/state/lib/state.dart (hunk 1 above)
Comment (hterkelsen) [inline modules/state/lib/state.dart]: btw, this could be written more compactly as:
```dart
List<Space> get values => _spaces.expand((x) => x);
```

Comment (jonahwilliams) [inline modules/state/lib/state.dart]: Done!

File: modules/ui/web/main.dart (hunk 2 above)
Comment (jonahwilliams) [inline modules/ui/web/main.dart]: Done!

**Pull request merged**